config/chime_science_run_gpu_no_output.yaml

```diff
@@ -0,0 +1,3456 @@
+      rfi_output_buf: gpu_rfi_output_buffer_3
+      rfi_mask_output_buf: gpu_rfi_mask_output_buffer_3
+      #rfi_var_output_buf: gpu_rfi_var_output_buffer_3
+
+# This set of stages takes information from the packet loss mask
+# and RFI mask and updates the metadata in the output N2 buffers
+# before they are read by other down stream stages.
+rfi_update_metadata:
+  update_metadata_0:
+    kotekan_stage: rfiUpdateMetadata
+    rfi_mask_buf: gpu_rfi_mask_output_buffer_0
+    lost_samples_buf: lost_samples_buffer
+    gpu_correlation_buf: gpu_n2_output_buffer_0
+  update_metadata_1:
+    kotekan_stage: rfiUpdateMetadata
+    rfi_mask_buf: gpu_rfi_mask_output_buffer_1
+    lost_samples_buf: lost_samples_buffer
+    gpu_correlation_buf: gpu_n2_output_buffer_1
+  update_metadata_2:
+    kotekan_stage: rfiUpdateMetadata
+    rfi_mask_buf: gpu_rfi_mask_output_buffer_2
+    lost_samples_buf: lost_samples_buffer
+    gpu_correlation_buf: gpu_n2_output_buffer_2
+  update_metadata_3:
+    kotekan_stage: rfiUpdateMetadata
+    rfi_mask_buf: gpu_rfi_mask_output_buffer_3
+    lost_samples_buf: lost_samples_buffer
+    gpu_correlation_buf: gpu_n2_output_buffer_3
+
+####  Gain update system for FRB and Pulsar ####
+read_gain:
+  updatable_config:
+    gain_frb: /updatable_config/frb_gain
+    gain_psr: /updatable_config/pulsar_gain
+  read_gain0:
+    kotekan_stage: ReadGain
+    in_buf: network_buffer_0
+    gain_frb_buf: gain_frb_buffer_0
+    gain_psr_buf: gain_psr_buffer_0
+  read_gain1:
+    kotekan_stage: ReadGain
+    in_buf: network_buffer_1
+    gain_frb_buf: gain_frb_buffer_1
+    gain_psr_buf: gain_psr_buffer_1
+  read_gain2:
+    kotekan_stage: ReadGain
+    in_buf: network_buffer_2
+    gain_frb_buf: gain_frb_buffer_2
+    gain_psr_buf: gain_psr_buffer_2
+  read_gain3:
+    kotekan_stage: ReadGain
+    in_buf: network_buffer_3
+    gain_frb_buf: gain_frb_buffer_3
+    gain_psr_buf: gain_psr_buffer_3
+
+#### FRB GPU Post processing and Tx ####
+frb:
+  factor_upchan_out: 16
+  num_beams_per_frb_packet: 4
+  timesamples_per_frb_packet: 16
+  cpu_affinity: [5,11]
+  frb_output_buffer:
+    num_frames: buffer_depth + 4
+    frame_size: 8 * 256 * (num_beams_per_frb_packet * num_gpus
+      * factor_upchan_out * timesamples_per_frb_packet
+      + 24 + sizeof_short * num_beams_per_frb_packet + sizeof_short * num_gpus
+      + sizeof_float * num_beams_per_frb_packet * num_gpus
+      + sizeof_float * num_beams_per_frb_packet * num_gpus)
+    metadata_pool: main_pool
+    kotekan_buffer: standard
+  postprocess:
+    log_level: warn
+    kotekan_stage: frbPostProcess
+    incoherent_beams: [0,256,512,768]
+    #incoherent_truncation: 25.
+    lost_samples_buf: lost_samples_buffer
+    frb_out_buf: frb_output_buffer
+    in_buf_0: gpu_beamform_output_buffer_0
+    in_buf_1: gpu_beamform_output_buffer_1
+    in_buf_2: gpu_beamform_output_buffer_2
+    in_buf_3: gpu_beamform_output_buffer_3
+    out_buf: frb_output_buffer
+  buffer_read:
```

Comment:
It seems weird that this stage is called `frb/buffer_read`. It's not very descriptive about what this process actually does. `frb/network_send` would make more sense. I appreciate the absurdity of making this comment in the file where it is actually disabled, but presumably it's also in the main config too.